### ISSUE TITLE
Composed Feature Models do not load changes that are made while they are closed

### ISSUE THREAD

**Issue by bjthehun** [FeatureIDE/FeatureIDE#1173]
#### Prerequisitives
* **Eclipse Version:**  2021-06 (4.20)
* **FeatureIDE-Version:** 3.7.2
* **Operating system:** Ubuntu 21.04
* **Java runtime used to run eclipse:** OpenJDK 64-Bit Server VM 16.0.1 64bit
* **FeatureIDE Composer:** Feature Modelling

### Issue description
When closing a composed UVL Feature Model, then changing a submodel it references, and saving those changes, the composed Feature Model does not load the changes and remains the same.

### Steps to reproduce
1. In the `Universal-Variability-Language-Model` example project, open both `OperatingSystem.uvl` and `Server.uvl`.
2. Close `Server.uvl`.
3. In `OperatingSystem.uvl`, select `macOS` and `Debian`, and create a feature above them named `Unix`.
4. Save `OperatingSystem.uvl.`
5. Open `Server.uvl`. The model remains unchanged.
6. Close and open `Server.uvl` again. Only now does the `Unix` feature appear.

**Comment by Rahel12342** [comment]
Isn't this the same issue as https://github.com/FeatureIDE/FeatureIDE/pull/1321 ?

**Comment by bjthehun** [comment]
No. #1321 is a pull request for Issue #1143, which was about what feature names are valid in UVL.
